models/campground.js

```diff
@@ -1,7 +1,6 @@
-// Import necessary modules
-const express = require('express');
-const router = express.Router();
-const Campground = require(''./review'');
```

Comment:
why did you delete these lines?
and the issue we have mentioned is still there.